.github/workflows/ci.yml

```diff
@@ -196,19 +196,16 @@ jobs:
           - i386
         include:
           - arch: amd64
-            CC: x86_64-pc-msys
-            toolchain: gcc
-            msystem: msys
+            msystem: UCRT64
```

Comment:
As we only ran windows tests for amd64 before i suspect the test failures might be some difference between msystem `MSYS` and `UCRT64`/`MINGW32`